laa-fee-calculator-client.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.6.0'
 
   spec.add_runtime_dependency 'addressable', '~> 2.3', '>= 2.3.7'
-  spec.add_runtime_dependency 'faraday', '>= 0.9.2', '< 0.18.0'
+  spec.add_runtime_dependency 'faraday', '>= 0.9.2', '< 1.4.0'
```

Comment:
looks like 1.3 is the [latest faraday on rubygems ](https://rubygems.org/gems/faraday/versions/0.9.2)

Reply:
That was taken from the dependabot Faraday pr https://github.com/ministryofjustice/laa-fee-calculator-client/pull/54. I think it will still evaulate to 1.3.

Reply:
less than 1.4 :doh: